Assets/Scripts/Visuals/UiHierarchyVisuals.cs

```diff
@@ -113,8 +113,15 @@ private void UpdateVisuals()
             }
             else
             {
-                MakeHierarchyItemsRecursive(scene, 0, scene.EntitiesInSceneRoot, mainPanelData);
-
+                if (scene.EntitiesInSceneRoot.Count() == 0)
+                {
+                    mainPanelData.AddTitle("No Entities");
```

Comment:
Please add the text, that the user should use right-click to add entities

Reply:
Like this? 
![image](https://user-images.githubusercontent.com/106346930/219670254-98c14265-6fdf-4647-b2a4-26d0ce4fa22b.png)